zds/tutorialv2/views/views_contents.py

```diff
@@ -726,37 +731,37 @@ def form_valid(self, form):
                 return super(UpdateContentWithArchive, self).form_invalid(form)
             else:
 
-                # warn user if licence have changed:
+                # Warn user if licence have changed:
                 manifest = json_reader.loads(unicode(zfile.read('manifest.json'), 'utf-8'))
                 if 'licence' not in manifest or manifest['licence'] != new_version.licence.code:
                     messages.info(
                         self.request, _(u'la licence « {} » a été appliquée.').format(new_version.licence.code))
 
-                # first, update DB object (in order to get a new slug if needed)
+                # First, update DB object (in order to get a new slug if needed).
                 self.object.title = new_version.title
                 self.object.description = new_version.description
                 self.object.licence = new_version.licence
-                self.object.type = new_version.type  # change of type is then allowed !!
+                self.object.type = new_version.type  # Change of type is then allowed !!
                 self.object.save()
 
-                new_version.slug = self.object.slug  # new slug if any !!
+                new_version.slug = self.object.slug  # New slug if any !!
 
-                # ok, then, let's do the import. First, remove everything in the repository
+                # ok, then, let's do the import. First, remove everything in the repository.
                 while True:
                     if versioned.children:
                         versioned.children[0].repo_delete(do_commit=False)
                     else:
-                        break  # this weird construction ensure that everything is removed
+                        break  # This weird construction ensure that everything is removed.
```

Comment:
Si ça s'assure que tout a été enlevé, ok. Si ça fait en sorte que tout sera supprimé, c'est pas `is` mais `gets`.